examples/bandwidthconstraints/src/index.js

```diff
@@ -217,19 +218,24 @@ function updateBandwidthParametersInRoom() {
   // media should join.
   roomName = someRoom.name;
 
-  // Attach the newly subscribed Track to the DOM and start the bitrate graph.
-  someRoom.on('trackSubscribed', attachTrack.bind(
-    null,
-    audioPreview,
-    videoPreview,
-    startAudioBitrateGraph.bind(null, someRoom),
-    startVideoBitrateGraph.bind(null, someRoom)));
-
-  // Detach the unsubscribed Track from the DOM and stop the bitrate graph.
-  someRoom.on('trackUnsubscribed', detachTrack.bind(
-    null,
-    audioPreview,
-    videoPreview));
+  // When a Participant joins the Room...
+  someRoom.on('participantConnected', function(participant) {
```

Comment:
@makarandp0 I don't think we need to change the examples. Here, we are focusing more on the specific use case, and not on Room setup.

Reply:
Okay, I'll revert change to this file as well.